blog/2022-12-6-benefits-challenges-localization.md

```diff
@@ -0,0 +1,87 @@
+
+Tolgee Chrome plugin may help you in the translation review phase to see how the final localized app will look like. Tolgee currently supports right-to-left (RTL) languages; however, we plan to make it even easier with a visual editor in Q1/2023. It will do the work with HTML characters and special characters in RTL languages a piece of cake. 
+
+
+## 3. Accurate translation
+
+Another challenge you may face when localizing your software is ensuring that all text is accurately translated. The best outcome you would certainly get is by hiring a professional translator. However, not all people want to make such an investment and prefer machine translations. However, machine translations are currently not at the stage when they can translate software solely on their own with perfect results. If you use machine translation and even if you choose [suitable MT provider](/blog/software-localization-machine-translation), you better ask a translator or native speaker to review the translations. You can learn how Tolgee can help you with MT translation in this [article.](/blog/machine-translation-in-tolgee)
+
+
+## 4. Lack of context of translations
+
+Context plays a crucial role in software localization and is necessary for accurate translation. Even if you hire the best translators, they can make mistakes if they don’t have all the needed information. Translators rely significantly on context as it helps them understand the meaning of the text they are translating. Unfortunately, translators still often receive software strings in spreadsheet form with zero context. To get accurate translations of your app, make sure to provide translators with the necessary context. Context is genuinely a [gamechanger in the software localization.](/blog/context-gamechanger-localization)
+
+First, Tolgee can offer context in the form of automatic screenshots to provide a visual context for the text that needs to be translated, making it easier to understand the meaning and intent. Second, an [in-context translation feature](/blog/in-context-production) can bring even more value to the whole localization process as it enables to translate directly in your app just with one click with all the context needed. 
+
+## 5. Formatting issues
+
+To make your software usable for customers in different regions, it’s not enough to simply translate the text. You must also account for formatting differences and conventions for writing dates, times, numbers, addresses, and currency. Hard-coded date, time, or currency formats will cause trouble during the localization process, as languages and countries use different date and time formats. 
+
+Tolgee deals with formatting issues using the [ICU (International Components for Unicode)](https://icu.unicode.org/home) message format. ICU also provides a lot of other helpful functionalities, such as advanced pluralization or selection rules. 
```

Comment:
"ICU message format". ICU is much more then just the format. Also, add link to our docs, not to the unicode page.